modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -167,6 +170,14 @@ public class ObjectEntryLocalServiceTest {
 			new LiferayIntegrationTestRule(),
 			SynchronousDestinationTestRule.INSTANCE);
 
+	@BeforeClass
+	public static void setUpClass() throws PortalException {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		_originalWorkflowAction = serviceContext.getWorkflowAction();
+	}
+
```

Comment:
Can you add this in `setUp`? If we add a `@BeforeClass`. We need create a `@AfterClass`.